src/components/scrim/edit-scrim-form.tsx

```diff
@@ -293,13 +314,32 @@ export function EditScrimForm({
           <FormItem>
             <FormLabel>Maps</FormLabel>
             <Accordion type="single" collapsible className="max-w-lg">
-              {maps.map((map) => (
+              {maps.map((map, index) => (
```

Comment:
This should be sorted by ID to make sure it is in a consistent order. When editing a map, it will cause it to jump around in the list. Go to the edit page and update where the `map` variable is declared to include a sort by ID:
```tsx
const maps = (
  await prisma.map.findMany({
    where: {
      scrimId: scrim.id,
    },
  })
).sort((a, b) => a.id - b.id);
```